std/array.d

```diff
@@ -588,6 +595,7 @@ if (isDynamicArray!T && allSatisfy!(isIntegral, I))
     return arrayAllocImpl!(true, T, ST)(sizes);
 }
 
+///
 @safe pure nothrow unittest
```

Comment:
This example is rather confusing, and seems to be testing edge cases. Can something more simple be used here?